sfputil/main.py

```diff
@@ -1991,11 +1993,48 @@ def loopback(port_name, loopback_mode):
         click.echo("{}: This functionality is not implemented".format(port_name))
         sys.exit(ERROR_NOT_IMPLEMENTED)
 
+    namespace = multi_asic.get_namespace_for_port(port_name)
+    config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+    if config_db is not None:
+        config_db.connect()
+        subport = int(config_db.get(config_db.CONFIG_DB,
+                                    'PORT|{}'.format(port_name), 'subport'))
+
+        # If subport is not defined (None) or set to 0, assign a default value of 1
+        # to ensure valid subport configuration.
+        if subport is None or subport == 0:
+            subport = 1
+    else:
+        click.echo("Failed to connect to CONFIG_DB")
+        sys.exit(EXIT_FAIL)
+
+    state_db = SonicV2Connector(use_unix_socket_path=False, namespace=namespace)
+    if state_db is not None:
+        state_db.connect(state_db.STATE_DB)
+        host_lane_count = int(state_db.get(state_db.STATE_DB,
```

Comment:
@xinyulin Wondering if we should handle ValueError exception and return error to handle case wherein `subport`, `host_lane_count` or `media_lane_count` are not present in the DB.

Reply:
@mihirpat1 It is helpful for the user to know about the incorrect configuration. The newly added commit catches the `TypeError` when the variable is not present in the DB. 
```python
try:
    subport = int(config_db.get(config_db.CONFIG_DB, f'PORT|{port_name}', 'subport'))
except TypeError:
    click.echo(f"{port_name}: subport is not present in CONFIG_DB")
    sys.exit(EXIT_FAIL)
``` 
From the formula, the `lane_mask` will automatically be set to 0 when the host or media lane count is zero. 
```python
if 'host-side' in loopback_mode:
    lane_mask = ((1 << host_lane_count) - 1) << ((subport - 1) * host_lane_count)
elif 'media-side' in loopback_mode:
    lane_mask = ((1 << media_lane_count) - 1) << ((subport - 1) * media_lane_count)
```